bundles/org.openhab.io.homekit/README.md

```diff
@@ -98,6 +99,7 @@ org.openhab.homekit:name=openHAB
 | networkInterface         | IP address or domain name under which the HomeKit bridge can be reached. If no value is configured, the add-on uses the first network adapter address configured for openHAB.                                                                                  | (none)        |
 | port                     | Port under which the HomeKit bridge can be reached.                                                                                                                                                                                     | 9123          |
 | useOHmDNS                | mDNS service is used to advertise openHAB as HomeKit bridge in the network so that HomeKit clients can find it. openHAB has already mDNS service running. This option defines whether the mDNS service of openHAB or a separate service should be used.   | false  |
+| blockUserDeletion        | Blocks HomeKit user deletion in openHAB and as result unpairing of devices. if you experience issue with accessories becoming non-responsive after some time, try to enable this setting. You can also enable this setting if you HomeKit setup is done and you will not re-pair ios devices.                                              | false         |
```

Comment:
... If you experience... 
... if your HomeKit...

Reply:
@lolodomo two typos in one sentence, argh :( thank you for pointing them out. fixed